week4/fuel.py

```diff
@@ -0,0 +1,17 @@
+
+
+fuel = ("Fraction: ")
```

Comment:
will this work? what is `("Fraction: ")`

Reply:
i dont think it works let me change it and recommit